src/test/java/org/opensearch/security/sanity/tests/SingleClusterSanityIT.java

```diff
@@ -34,6 +38,13 @@ public void testSecurityPluginInstallation() throws Exception {
         verifyPluginInstallationOnAllNodes();
     }
 
+    @Test
+    public void testAdminCredentials_customStrongPassword_shouldSucceed() throws Exception {
+        Response response = client().performRequest(new Request("GET", ""));
```

Comment:
This test doesn't set or verify the strength of the password, can we change the name or make the test assert something about the state of the system?

Reply:
the purpose of this test is to verify that admin is called with correct credentials and those correct credentials are assumed to have strong password which was used during cluster formation.